pkg/filter/driftignore.go

```diff
@@ -143,6 +152,51 @@ RuleCheck:
 	return false
 }
 
+func AnalysisToList(analysis *analyser.Analysis, opts AnalysisListOptions) (int, string) {
```

Comment:
Why putting this method there ? It's related to `Analysis` and maybe we can make it an Analysis struct method like `analysis.ToDriftIgnoreList()`

You can see it as another way to serialize an analysis struct. It should not remain in the filter pkg for me.